### ISSUE TITLE
task_module doesn't output all files

### ISSUE THREAD

**Issue by bdeck8317** [PennBBL/xcpEngine#379]
**Describe the bug**
I am attempting to obtain task regressed data from a series of functional images where there are multiple tasks and multiple runs of the same task. The behavior is odd, as the images are created in the task folder but then are deleted. I have the task module on rerun so that I can regress out the task activity. Additionally, the report is not generated. 

**Cohort file**

```
id0,id1,id2,id3,img,task_design
sub-DP5001,ses-NavonRealStim,NavonLow,run-1,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/design_stroop_withwmcsfreg.fsf
sub-DP5001,ses-NavonRealStim,NavonLow,run-2,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/design_stroop_withwmcsfreg.fsf
sub-DP5001,ses-NavonRealStim,NavonHigh,run-1,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonHigh_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/design_stroop_withwmcsfreg.fsf
sub-DP5001,ses-NavonRealStim,NavonHigh,run-2,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonHigh_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/design_stroop_withwmcsfreg.fsf

```

**Design File**

```
#!/usr/bin/env bash

###################################################################
#  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  �^�^�  #
###################################################################


###################################################################
# This design file stores the values of all variables required to
# execute a complete neuroimage processing pipeline. You may
# execute the analysis specified in this design file by calling
# (in any v4 or higher bash terminal):
#
# xcpEngine <file>.dsn
#
# Variables fall into five general categories:
# * ANALYSIS VARIABLES are used at all stages of this analysis.
# * PIPELINE specifies the modules that comprise the analysis.
# * MODULE VARIABLES are used during one stage of the analysis.
#                  These are typically array variables with array
#                  indices equal to the index of the module that
#                  calls them.
# * OUTPUT VARIABLES may be used at all stages of the analysis.
#                  These are typically array variables with array
#                  indices equal to the value of the primary
#                  subject identifier. They will appear only in
#                  localised design files.
###################################################################


###################################################################
# ANALYSIS VARIABLES
###################################################################

analysis=task_$(whoami)
design=/data/xcp_git_repo/xcpEngine/designs/task.dsn
sequence=anatomical
standard=MNI%2x2x2

###################################################################
# PIPELINE
###################################################################

pipeline=task,roiquant,norm
###################################################################
# 1 TASK
###################################################################

task_fmriprep[1]=1
task_vol2del[1]=0
task_confound[1]='36p'
task_rerun[1]=Y
task_cleanup[1]=Y

###################################################################
# 2 ROIQUANT
###################################################################

roiquant_atlas[2]=all
roiquant_globals[2]=1
roiquant_vol[2]=0
roiquant_rerun[2]=1
roiquant_cleanup[2]=1


###################################################################
# 3 NORM
###################################################################

norm_rerun[3]=1
norm_cleanup[3]=1


```
**Error message**

```
1.) Files are present and then are gone: 

(base) neurouser1@heph task % ls -lth
total 25232
drwxr-xr-x  3 neurouser1  staff    96B Aug 20 15:17 model
-rw-r--r--  1 neurouser1  staff   504K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_confmat.1D
-rw-r--r--  1 neurouser1  staff    16K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_wm.1D
-rw-r--r--  1 neurouser1  staff    16K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_gsr.1D
-rw-r--r--  1 neurouser1  staff    16K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_csf.1D
-rw-r--r--  1 neurouser1  staff    10B Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregDice.txt
-rw-r--r--  1 neurouser1  staff    10B Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregJaccard.txt
-rw-r--r--  1 neurouser1  staff    10B Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregCoverage.txt
-rw-r--r--  1 neurouser1  staff    10B Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregCrossCorr.txt
-rw-r--r--  1 neurouser1  staff   200K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_structbrain.nii.gz
-rw-r--r--  1 neurouser1  staff   186K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_referenceVolumeBrain.nii.gz
-rw-r--r--  1 neurouser1  staff   574K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_struct.nii.gz
-rw-r--r--  1 neurouser1  staff    18K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_segmentation.nii.gz
-rw-r--r--  1 neurouser1  staff   6.7K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_mask.nii.gz
-rw-r--r--  1 neurouser1  staff   436K Aug 20 15:17 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_referenceVolume.nii.gz
drwxr-xr-x  5 neurouser1  staff   160B Aug 20 15:16 mc
-rw-r--r--  1 neurouser1  staff    58K Aug 20 15:16 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_motion.1D
-rw-r--r--  1 neurouser1  staff   482K Aug 20 15:16 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_fmriconf.json
-rw-r--r--  1 neurouser1  staff   9.8M Aug 20 15:16 sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_fmriconf.tsv


second 'ls'

(base) neurouser1@heph task % ls
mc                                                              sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregDice.txt      sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_gsr.1D
model                                                           sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregJaccard.txt   sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_motion.1D
sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_confmat.1D         sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_csf.1D             sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_wm.1D
sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregCoverage.txt  sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_fmriconf.json
sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_coregCrossCorr.txt sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_fmriconf.tsv


2.) Error generating report


Module complete
generating report
Traceback (most recent call last):
  File "/xcpEngine/core/report.py", line 130, in <module>
    himg = load_img(outdir+'/task/'+prefix+'_processed.nii.gz').header
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/image/image.py", line 1063, in load_img
    return check_niimg(img, wildcards=wildcards, dtype=dtype)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/_utils/niimg_conversions.py", line 251, in check_niimg
    raise ValueError("File not found: '%s'" % niimg)
ValueError: File not found: '/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_processed.nii.gz'
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
Traceback (most recent call last):
  File "/xcpEngine/core/report.py", line 130, in <module>
    himg = load_img(outdir+'/task/'+prefix+'_processed.nii.gz').header
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/image/image.py", line 1063, in load_img
    return check_niimg(img, wildcards=wildcards, dtype=dtype)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/_utils/niimg_conversions.py", line 251, in check_niimg
    raise ValueError("File not found: '%s'" % niimg)
ValueError: File not found: '/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-2/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-2_processed.nii.gz'
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
Traceback (most recent call last):
  File "/xcpEngine/core/report.py", line 130, in <module>
    himg = load_img(outdir+'/task/'+prefix+'_processed.nii.gz').header
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/image/image.py", line 1063, in load_img
    return check_niimg(img, wildcards=wildcards, dtype=dtype)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/_utils/niimg_conversions.py", line 251, in check_niimg
    raise ValueError("File not found: '%s'" % niimg)
ValueError: File not found: '/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonHigh/run-1/task/sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_processed.nii.gz'
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
Traceback (most recent call last):
  File "/xcpEngine/core/report.py", line 130, in <module>
    himg = load_img(outdir+'/task/'+prefix+'_processed.nii.gz').header
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/image/image.py", line 1063, in load_img
    return check_niimg(img, wildcards=wildcards, dtype=dtype)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/_utils/niimg_conversions.py", line 251, in check_niimg
    raise ValueError("File not found: '%s'" % niimg)
ValueError: File not found: '/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonHigh/run-2/task/sub-DP5001_ses-NavonRealStim_NavonHigh_run-2_processed.nii.gz'
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
Traceback (most recent call last):


Traceback (most recent call last):
  File "/xcpEngine/core/report.py", line 130, in <module>
    himg = load_img(outdir+'/task/'+prefix+'_processed.nii.gz').header
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/image/image.py", line 1063, in load_img
    return check_niimg(img, wildcards=wildcards, dtype=dtype)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nilearn/_utils/niimg_conversions.py", line 251, in check_niimg
    raise ValueError("File not found: '%s'" % niimg)
ValueError: File not found: '/data/xcpOutput/task_data/sub-DP5001/ses-StroopShamStim/StroopHigh/run-2/task/sub-DP5001_ses-StroopShamStim_StroopHigh_run-2_processed.nii.gz'
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit



###################################################################
#  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  #
#                                                                 #
#  �                        TERMINATOR                         �  #
#                                                                 #
#  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  #

```

**Runtime Information**
Docker command:
docker run -it \
    -v /Users/neurouser1/Documents/DP5/xcp/:/data \
    pennbbl/xcpengine:latest \
    -c /data/cohorts/DP5_task_cohort.csv \
    -d /data/xcp_git_repo/xcpEngine/designs/task.dsn \
    -o /data/xcpOutput/task_data/ \
    -i /data/xcpOutput/task_data/tmp/ \
    -t 1 \
    -r /data/fmriprep_out/fmriprep/

**Additional context**

**Comment by bdeck8317** [comment]
@a3sha2 
Thanks for looking over this issue.

I am still not seeing the following directories. 


```
task/fsl # the fsl feat directory
task/model # the supplied design files and model
task/logs # logs and html report
task/mc # motion parameters
task/sigchange # % signal change computed from pes

```

Any other reason this could be?

**Comment by a3sha2** [comment]
pls share the log files with me

**Comment by bdeck8317** [comment]
@a3sha2 

Here are the logs

[task_root_202008231444_sub-DP5001_ses-NavonRealStim_NavonHigh_run-1LOG-process.txt](https://github.com/PennBBL/xcpEngine/files/5118120/task_root_202008231444_sub-DP5001_ses-NavonRealStim_NavonHigh_run-1LOG-process.txt)
[task_root_202008231444_LOG-process.txt](https://github.com/PennBBL/xcpEngine/files/5118121/task_root_202008231444_LOG-process.txt)

**Comment by a3sha2** [comment]
feat didnt run , there is likely issue with task *fsf

**Comment by bdeck8317** [comment]
@a3sha2 , right. 

```

###################################################################
#  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  #
#                                                                 #
#  ⊗            TASK-CONSTRAINED ACTIVATION MODULE             ⊗  #
#                                                                 #
#  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  ⊗  #
###################################################################



[I][/data/xcpOutput/task_data/sub-DP5001/ses-StroopRealStim/StroopLow/run-1/sub-DP5001_ses-StroopRealStim_StroopLow_run-1.nii.gz]
[O][/data/xcpOutput/task_data/sub-DP5001/ses-StroopRealStim/StroopLow/run-1/task]



Current processing step:
Localising FEAT design
····································································
/xcpEngine/core/functions/printx: line 2: ${@}: ambiguous redirect
** AFNI converts NIFTI_datatype=4 (INT16) in file /data/xcpOutput/task_data/sub-DP5001/ses-StroopRealStim/StroopLow/run-1/sub-DP5001_ses-StroopRealStim_StroopLow_run-1.nii.gz to FLOAT32
     Warnings of this type will be muted for this session.
     Set AFNI_NIFTI_TYPE_WARN to YES to see them all, NO to see none.
NATIVE:LPI TEMPLATE:LPI
· NOT re-orienting native



Current processing step:
getting data from frmiprep directory
····································································
mkdir: cannot create directory ‘/data/xcpOutput/task_data/sub-DP5001/ses-StroopRealStim/StroopLow/run-1/task/mc’: File exists
· relative maximum motion
· relative mean motion

                   The template is MNI
                   xcpEngine only support MNI, PNC and OASIS templates for now

· generate new /data/xcpOutput/task_data/sub-DP5001/ses-StroopRealStim/StroopLow/run-1/sub-DP5001_ses-StroopRealStim_StroopLow_run-1_spaces.json with spaceMetadata
· Quality assessment
· [Cross-correlation:    0.9221351 ]
· [Coverage:             0.9886942 ]
· [Jaccard coefficient:  0.9000222 ]
· [Dice coefficient:     0.9473807 ]
····································································
Processing step complete:
getting data from frmiprep directory
· Importing analysis variables
· Parsing design and contrasts
· generate confound
/xcpEngine/modules/task/task.mod: line 820: ((: == 1 : syntax error: operand expected (error token is "== 1 ")




Current processing step:
Executing FEAT analysis
····································································
· Preparing environment
· Processing FEAT design:
· /data/xcpOutput/task_data/sub-DP5001/ses-StroopRealStim/StroopHigh/run-1/task/model/sub-DP5001_ses-StroopRealStim_StroopHigh_run-1_design.fsf
invalid command name "toplevel"
    while executing
"toplevel .prompt -borderwidth 5"
    (procedure "MxPause" line 4)
    invoked from within
"MxPause "FEAT setup file is too old to load - sorry!""
    (procedure "feat5:load" line 74)
    invoked from within
"feat5:load -1 1 ${fsfroot}.fsf"
    (file "/opt/fsl-5.0.10/bin/feat" line 309)
····································································
Processing step complete:
Executing FEAT analysis


Module complete


```

Here is the feat output.

**Comment by bdeck8317** [comment]
And More verbose


```
###################################################################
#  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  #
#                                                                 #
#  �            TASK-CONSTRAINED ACTIVATION MODULE             �  #
#                                                                 #
#  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  �  #
###################################################################



[I][/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1.nii.gz]
[O][/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task]
                             -   -   -
>> rln /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1.nii.gz /data/xcpOutput/task_data/tmp/sub-DP5001_ses-NavonRealStim_NavonLow_run-1-task-995816503~TEMP~.nii.gz
                             -   -   -
>> 5-3-1
                             -   -   -
>> 5-1



Current processing step:
@1 Localising FEAT design
····································································
                             -   -   -
>> mkdir -p /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/model
/xcpEngine/core/functions/printx: line 2: ${@}: ambiguous redirect
** AFNI converts NIFTI_datatype=4 (INT16) in file /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1.nii.gz to FLOAT32
     Warnings of this type will be muted for this session.
     Set AFNI_NIFTI_TYPE_WARN to YES to see them all, NO to see none.
NATIVE:LPI TEMPLATE:LPI
· @0.1d NOT re-orienting native



Current processing step:
@ getting data from frmiprep directory
····································································
                             -   -   -
>> cp /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-1_desc-confounds_regressors.tsv /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv
                             -   -   -
>> cp /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-1_desc-confounds_regressors.json /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.json
                             -   -   -
>> mkdir /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc
mkdir: cannot create directory �/data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc�: File exists
                             -   -   -
>> ${XCPEDIR}/utils/generate_confmat.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv -j rps -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_motion.1D
                             -   -   -
>> ${XCPEDIR}/utils/generate_confmat.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv -j fd -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_relRMS.1D
· @1.2 relative maximum motion
                             -   -   -
>> ${XCPEDIR}/utils/1dTool.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_relRMS.1D -o max -f /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_relMaxRMS.txt
· @1.3 relative mean motion
                             -   -   -
>> ${XCPEDIR}/utils/1dTool.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_relRMS.1D -o mean -f /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/mc/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_relMeanRMS.txt

                   The template is MNI
                   xcpEngine only support MNI, PNC and OASIS templates for now

                             -   -   -
>> 3dresample -orient LPI -inset /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-1_space-T1w_boldref.nii.gz -prefix /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolume.nii.gz -overwrite
                             -   -   -
>> 3dresample -orient LPI -inset /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-1_space-T1w_desc-brain_mask.nii.gz -prefix sub-DP5001_ses-NavonRealStim_NavonLow_run-1_imgmask.nii.gz -overwrite
                             -   -   -
>> 3dresample -master /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolume.nii.gz -inset /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/..//anat/sub-DP5001_desc-brain_mask.nii.gz -prefix /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_structmask.nii.gz -overwrite
                             -   -   -
>> fslmaths sub-DP5001_ses-NavonRealStim_NavonLow_run-1_imgmask.nii.gz -mul /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_structmask.nii.gz /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_mask.nii.gz
                             -   -   -
>> 3dresample -master /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolume.nii.gz -inset /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/..//anat/sub-DP5001_dseg.nii.gz -prefix /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_segmentation.nii.gz -overwrite
                             -   -   -
>> 3dresample -master /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolume.nii.gz -inset /data/fmriprep_out/fmriprep/sub-DP5001/ses-NavonRealStim/..//anat/sub-DP5001_desc-preproc_T1w.nii.gz -prefix /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_struct.nii.gz -overwrite
                             -   -   -
>> fslmaths /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_mask.nii.gz -mul /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolume.nii.gz /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolumeBrain.nii.gz
                             -   -   -
>> fslmaths /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_mask.nii.gz -mul /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_struct.nii.gz /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_structbrain.nii.gz
                             -   -   -
>> rm -rf /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_structmask.nii.gz sub-DP5001_ses-NavonRealStim_NavonLow_run-1_imgmask.nii.gz
· @ generate new /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_spaces.json with spaceMetadata
· @ Quality assessment
                             -   -   -
>> ${XCPEDIR}/utils/maskOverlap.R -m /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_segmentation.nii.gz -r /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolumeBrain.nii.gz
· [Cross-correlation:    0.9194825 ]
· [Coverage:             0.9848067 ]
· [Jaccard coefficient:  0.8969894 ]
· [Dice coefficient:     0.9456978 ]
····································································
Processing step complete:
getting data from frmiprep directory
· @1.1a Importing analysis variables
· @1.1b Parsing design and contrasts
                             -   -   -
>> fslval /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1.nii.gz pixdim4
                             -   -   -
>> fslval /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1.nii.gz dim4
· @1.1 generate confound
                             -   -   -
>> ${XCPEDIR}/utils/mbind.R -y /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_motion.1D -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D
                             -   -   -
>> ${XCPEDIR}/utils/generate_confmat.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv -j csf -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_csf.1D
                             -   -   -
>> ${XCPEDIR}/utils/mbind.R -x /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D -y /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_csf.1D -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D
                             -   -   -
>> ${XCPEDIR}/utils/generate_confmat.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv -j gsr -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_gsr.1D
                             -   -   -
>> ${XCPEDIR}/utils/mbind.R -x /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D -y /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_gsr.1D -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D
                             -   -   -
>> ${XCPEDIR}/utils/generate_confmat.R -i /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv -j wm -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_wm.1D
                             -   -   -
>> ${XCPEDIR}/utils/mbind.R -x /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D -y /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_wm.1D -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D
                             -   -   -
>> ${XCPEDIR}/utils/mbind.R -x /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D -y OPdx1 -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D
                             -   -   -
>> ${XCPEDIR}/utils/mbind.R -x /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D -y OPpower2 -o /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D
/xcpEngine/modules/task/task.mod: line 820: ((: == 1 : syntax error: operand expected (error token is "== 1 ")
                             -   -   -
>> rm -f /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/model/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_design.fsf



Current processing step:
@2 Executing FEAT analysis
····································································
· @2.1 Preparing environment
                             -   -   -
>> rm -rf /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/fsl/sub-DP5001_ses-NavonRealStim_NavonLow_run-1.feat
· @2.2a Processing FEAT design:
· @2.2b /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/model/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_design.fsf
                             -   -   -
>> feat /data/xcpOutput/task_data/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/model/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_design.fsf
invalid command name "toplevel"
    while executing
"toplevel .prompt -borderwidth 5"
    (procedure "MxPause" line 4)
    invoked from within
"MxPause "FEAT setup file is too old to load - sorry!""
    (procedure "feat5:load" line 74)
    invoked from within
"feat5:load -1 1 ${fsfroot}.fsf"
    (file "/opt/fsl-5.0.10/bin/feat" line 309)
····································································
Processing step complete:
Executing FEAT analysis



Module Workflow Map
····································································
· START
· @1
· @0.1d
· @
· @1.2
· @1.3
· @
· @
· @1.1a
· @1.1b
· @1.1
· @2
· @2.1
· @2.2a
· @2.2b
· FINISH
····································································


Module complete
```

**Comment by a3sha2** [comment]
send me on you fsf file

**Comment by bdeck8317** [comment]
@a3sha2 
Sure, here you go.

[design_stroop_nowmcsfreg.txt](https://github.com/PennBBL/xcpEngine/files/5118786/design_stroop_nowmcsfreg.txt)

**Comment by a3sha2** [comment]
no event file in fsf file

**Comment by bdeck8317** [comment]
@a3sha2 , 

I have created a new .fsf file 
[task.txt](https://github.com/PennBBL/xcpEngine/files/5137186/task.txt)

However, I am encountering the same issue as before where feat is not running. 

Further insight to this is that the model folder inside the task directory has the following file inside
```sub-DP5001_ses-NavonRealStim_NavonHigh_run-1_design.fsf
```
------This file is empty. Does this point to an issue with the cohort file?

```
id0,id1,id2,id3,img,task_design
sub-DP5001,ses-NavonRealStim,NavonLow,run-1,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-NavonRealStim,NavonLow,run-2,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonLow_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-NavonRealStim,NavonHigh,run-1,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonHigh_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-NavonRealStim,NavonHigh,run-2,sub-DP5001/ses-NavonRealStim/func/sub-DP5001_ses-NavonRealStim_task-NavonHigh_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopRealStim,StroopLow,run-1,sub-DP5001/ses-StroopRealStim/func/sub-DP5001_ses-StroopRealStim_task-StroopLow_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopRealStim,StroopLow,run-2,sub-DP5001/ses-StroopRealStim/func/sub-DP5001_ses-StroopRealStim_task-StroopLow_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopRealStim,StroopHigh,run-1,sub-DP5001/ses-StroopRealStim/func/sub-DP5001_ses-StroopRealStim_task-StroopHigh_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopRealStim,StroopHigh,run-2,sub-DP5001/ses-StroopRealStim/func/sub-DP5001_ses-StroopRealStim_task-StroopHigh_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopShamStim,StroopLow,run-1,sub-DP5001/ses-StroopShamStim/func/sub-DP5001_ses-StroopShamStim_task-StroopLow_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopShamStim,StroopLow,run-2,sub-DP5001/ses-StroopShamStim/func/sub-DP5001_ses-StroopShamStim_task-StroopLow_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopShamStim,StroopHigh,run-1,sub-DP5001/ses-StroopShamStim/func/sub-DP5001_ses-StroopShamStim_task-StroopHigh_run-1_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
sub-DP5001,ses-StroopShamStim,StroopHigh,run-2,sub-DP5001/ses-StroopShamStim/func/sub-DP5001_ses-StroopShamStim_task-StroopHigh_run-2_space-T1w_desc-preproc_bold.nii.gz,task_designs/task.fsf
```


Let me know your thoughts.

**Comment by bdeck8317** [comment]
@a3sha2 , I am going to close this issue. I had a number of issues with the .fsf file which you pointed out previously. I also had an issue with the directory which my .fsf file was located. I have fixed these issues and the task regression is going well! 


Thank you so much for your patience and assistance!

**Comment by bdeck8317** [comment]
@a3sha2 ,

It seems that all the proper output files are present with the exception of the ```res4d_lh/rh``` files.

Docker command (includes path to fmriprep output folder): 

```
docker run -it \
    -v /Users/neurouser1/Documents/DP5/xcp/:/data \
    pennbbl/xcpengine:latest \
    -c /data/cohorts/DP5_task_cohort.csv \
    -d /data/task_designs/task_fcon.dsn \
    -o /data/xcpOutput/task_data/test \
    -i /data/xcpOutput/task_data/test/tmp/ \
    -r /data/fmriprep_out/
```

```
copes                                                                   sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.json
fsl                                                                     sub-DP5001_ses-NavonRealStim_NavonLow_run-1_fmriconf.tsv
logs                                                                    sub-DP5001_ses-NavonRealStim_NavonLow_run-1_gsr.1D
mc                                                                      sub-DP5001_ses-NavonRealStim_NavonLow_run-1_mask.nii.gz
model                                                                   sub-DP5001_ses-NavonRealStim_NavonLow_run-1_meanIntensity.nii.gz
pes                                                                     sub-DP5001_ses-NavonRealStim_NavonLow_run-1_meanIntensityBrain.nii.gz
sigchange                                                               sub-DP5001_ses-NavonRealStim_NavonLow_run-1_motion.1D
stats                                                                   sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolume.nii.gz
sub-DP5001_ses-NavonRealStim_NavonLow_run-1_confmat.1D                  sub-DP5001_ses-NavonRealStim_NavonLow_run-1_referenceVolumeBrain.nii.gz
sub-DP5001_ses-NavonRealStim_NavonLow_run-1_coregCoverage.txt           sub-DP5001_ses-NavonRealStim_NavonLow_run-1_segmentation.nii.gz
sub-DP5001_ses-NavonRealStim_NavonLow_run-1_coregCrossCorr.txt          sub-DP5001_ses-NavonRealStim_NavonLow_run-1_struct.nii.gz
sub-DP5001_ses-NavonRealStim_NavonLow_run-1_coregDice.txt               sub-DP5001_ses-NavonRealStim_NavonLow_run-1_structbrain.nii.gz
sub-DP5001_ses-NavonRealStim_NavonLow_run-1_coregJaccard.txt            sub-DP5001_ses-NavonRealStim_NavonLow_run-1_wm.1D
sub-DP5001_ses-NavonRealStim_NavonLow_run-1_csf.1D                      varcopes

```


Not sure what I am missing here. My understanding is that if you include the freesurfer directory within the fmriprep directory I should obtain these task regressed task residualized images. 

Thoughts?

**Comment by a3sha2** [comment]
if fmriprep output include freesurfer, xcp will write out surface files for resd4D

**Comment by a3sha2** [comment]
that was the reason, `/xcpEngine/modules/task/task.mod: line 1140: ${1wtotemp2}: bad substitution`

**Comment by bdeck8317** [comment]
@a3sha2 , Cool will pull down once the build is complete and re-run. Thanks so much for your help!

**Comment by a3sha2** [comment]
it is done, you can pull for new one!

**Comment by bdeck8317** [comment]
@a3sha2 , 

Thanks again for helping with previous issues.

I found another issue with ANTS on the ```/xcpEngine/core/functions/exec_ants``` function. 


Here is the entire error: 

```
· generate new /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_spaces.json with spaceMetadata
· Quality assessment
· [Cross-correlation:    0.9194825 ]
· [Coverage:             0.9848067 ]
· [Jaccard coefficient:  0.8969894 ]
· [Dice coefficient:     0.9456978 ]
····································································
Processing step complete:
getting data from frmiprep directory
· Importing analysis variables
· Parsing design and contrasts
· generate confound
· Importing confounds



Current processing step:
Executing FEAT analysis
····································································
· Preparing environment
· Processing FEAT design:
· /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/model/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_design.fsf
····································································
Processing step complete:
Executing FEAT analysis



Current processing step:
Reorganising FEAT output
····································································
· Extracting mean brain
· Re-localising model design
····································································
Processing step complete:
Reorganising FEAT output



Current processing step:
Processing parameter estimates
····································································
· Obtaining peak magnitudes: PEs
· Raw parameter estimates
· Obtaining peak magnitudes: contrasts
· Contrasts
· cross
· task
· end_cross
/xcpEngine/core/functions/exec_ants: line 1:  4112 Killed                  ${ANTSPATH}/"${@}"
srcvol = /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/boldtoMNI.nii.gz
srcreg unspecified
srcregold = 0
srcwarp unspecified
surf = white
hemi = lh
ProjFrac = 0.5
thickness = thickness
reshape = 0
interp = nearest
float2int = round
GetProjMax = 0
INFO: float2int code = 0
niiRead(): error opening file /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/boldtoMNI.nii.gz
ERROR: could not read /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/boldtoMNI.nii.gz as type 24
niiRead(): error opening file /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/lh_surface.nii.gz
ERROR: unable to to read /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/lh_surface.nii.gz
srcvol = /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/boldtoMNI.nii.gz
srcreg unspecified
srcregold = 0
srcwarp unspecified
surf = white
hemi = rh
ProjFrac = 0.5
thickness = thickness
reshape = 0
interp = nearest
float2int = round
GetProjMax = 0
INFO: float2int code = 0
niiRead(): error opening file /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/boldtoMNI.nii.gz
ERROR: could not read /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/boldtoMNI.nii.gz as type 24
niiRead(): error opening file /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/rh_surface.nii.gz
ERROR: unable to to read /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/rh_surface.nii.gz

While running:
/xcpEngine/thirdparty/workbench/bin_rh_linux64/../exe_rh_linux64/wb_command -cifti-create-dense-scalar /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_res4D.dscalar.nii -left-metric /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_res4d_lh.func.gii -right-metric /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task/sub-DP5001_ses-NavonRealStim_NavonLow_run-1_res4d_rh.func.gii

ERROR: NAME OF FILE: sub-DP5001_ses-NavonRealStim_NavonLow_run-1_res4d_lh.func.gii
PATH TO FILE: /data/xcpOutput/task_data/test/sub-DP5001/ses-NavonRealStim/NavonLow/run-1/task

File does not exist.

····································································
Processing step complete:
Processing parameter estimates


Module complete

```


Let me know if this is an error on my side or another bug.

Thanks,
Ben

**Comment by a3sha2** [comment]
/xcpEngine/core/functions/exec_ants: line 1:  4112 Killed    that is memory issue

**Comment by bdeck8317** [comment]
@a3sha2 , ohh wow. That's surprising. 
![image](https://user-images.githubusercontent.com/49119525/91897451-4f55d300-ec68-11ea-9fd6-f418716bbf5e.png)

Based on these preferences with 27gb of memory allocated and 27 cpu's. I am allowing the docker container to use all of this pre-allocated space. 

Is there a way to ensure this memory issue doesn't occur?

Ben

**Comment by bdeck8317** [comment]
@a3sha2 , the task module ran properly but I have a couple questions about configuring the docker container and the minimum specifications. I will close this and open a new issue.
